### PR TITLE
Store the chain data and keys outside of the repo dir

### PR DESCRIPTION
**Pull request by piotrMocz**
This means when the user deletes their repository and re-clones, the session keys will stay intact (as will the db backup, which is also quite handy).

You can configure the directory where the data will be stored by using the `--data_dir` option. It defaults to `${HOME}/.alephzero` but I'm not dead set on this particular value :)

### CODE REVIEW

File: run_node.sh [hunk 1]
```diff
@@ -26,7 +26,7 @@ Help()
 }
 
 echo "Updating this repo..."
-git pull origin main
+# git pull origin main
```
Comment (maciejnems) [inline run_node.sh:29]: This comment is added here by accident, right?

File: run_node.sh [hunk 2]
```diff
@@ -26,7 +26,7 @@ Help()
 }
 
 echo "Updating this repo..."
-git pull origin main
```
Comment (maciejnems) [inline run_node.sh:28]: So right now user would need to run this script twice, to update this file. Maybe we should remove completely this step of updating repo and say to users to update it manually? I'm not sure whether our first idea of including this step in run_node script was correct

File: run_node.sh [hunk 3]
```diff
@@ -77,6 +82,28 @@ done
 
 ALEPH_VERSION=$(cat env/version)
 
+if [ ! -d "${HOST_BASE_PATH}/${DB_SNAPSHOT_PATH}" ] && [ -d "${DB_SNAPSHOT_PATH}/keystore" ]
+then
+    echo "The default location of the data directory has changed."
+    echo "Your files will be copied automatically to ${HOST_BASE_PATH}/${DB_SNAPSHOT_PATH}."
+    echo "If you wish to customize the directory, select \'n\' and re-run the script"
```
Comment (maciejnems) [inline run_node.sh:89]: Here it could be just `'n'`, without `\` same for other two places with `--data_dir`

File: signer.sh [hunk 4]
```diff
@@ -26,7 +26,8 @@ CONTAINER_NAME="aleph-signer"
 SIGNER_VERSION="fca5dd5"
 NODE_KEY_PATH="/data/p2p_secret"
 SIGNER_IMAGE=public.ecr.aws/p6e8q1z1/peer-verifier:${SIGNER_VERSION}
-HOST_SECRET_PATH="$(pwd)/p2p_secret"
+ALEPH_DIRECTORY="${HOME}/.alephzero"
```
Comment (fbielejec) [inline signer.sh:29]: that seems to be the default, but what if it was overridden (as per -d | --data_dir argument to th enode runner script)?

Comment (piotrMocz) [reply signer.sh:29]: Well, then it would make this whole operation a no-op, which is not the end of the world (although doesn't make much sense)

**Review by maciejnems — APPROVED** [review]

**Pull request merged**